mycity/mycity/intents/user_address_intent.py

```diff
@@ -22,6 +24,45 @@ def set_address_in_session(mycity_request):
             mycity_request.intent_variables['Address']['value']
 
 
+def get_address_from_user_device(mycity_request):
+    """
+    checks Amazon api for device address permissions. 
+    If given, the address, if present, will be stored 
+    in the session attributes, otherwise the user will 
+    be notified that no address is present.

```

Comment:
I would remove the last part (from `, otherwise` to the end) because this function doesn't provide that notification (it happens in the get_address_intent when an address is not found in the session attributes).